db/migrate/20171124131432_create_registries.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRegistries < ActiveRecord::Migration
```

Comment:
I'd rather modify the last migration, since if we go this way this PR is missing two migrations: one to remove the old table, and another one to migrate the SUSE registry data from the `pillars` table and removing it from there.

The first migration I think could be avoided by just modifying the `CreateDockerRegistries` migration (after all, this hasn't been released), and we need an extra migration for the `Pillar` to first-class citizen migration of the SUSE registry mirror.